portingdb/templates/index.html

```diff
@@ -4,7 +4,9 @@
 {% endblock breadcrumbs %}
 
 {% block titlecontent -%}
-    {{ (the_score * 100) | round(1) }}% – {{ super() }}
+    {{ (the_score * 100) | round(1) }}% →
+    {{ (py2_score * 100) | round(1) }}% –
+    {{ super() }}
```

Comment:
The new title is a bit comprehensive to grasp.

Reply:
So, I tried explaining it on the main page.